ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/services/individual-model-mapper.ts

```diff
@@ -28,7 +35,13 @@ export class IndividualModelMapper extends ParticipantModelMapper {
         model.interpreter = this.mapBooleanValue(response.answers, IndividualQuestionKeys.Interpreter);
         model.computer = this.mapBooleanValue(response.answers, IndividualQuestionKeys.Computer);
         model.camera = this.mapComputerCamera(response.answers);
+
+        // Self Test
```

Comment:
could maybe move this up to the `ParticipantModelMapper` base class since it'll be the same for representative as well?

is there any test covering this mapping btw?

Reply:
If I move it up,I get angular injection problems.. ?

There is test coverage